daprdocs/content/en/concepts/components-concept.md

```diff
@@ -19,6 +19,8 @@ As another example, the [pub/sub]({{< ref "pubsub-overview.md" >}}) building blo
 
 You can get a list of current components available in the hosting environment using the `dapr components` CLI command.
 
+*Note: for components that read data, it is recommended to set the memory capacity of the Dapr process/container accordingly to avoid potential OOM panics*
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}} 
For components that read data, it is recommended to set appropriate memory capacity of the Dapr sidecar (process or container) to avoid potential OOM panics. For example in Docker use the `--memory` option. For process this depend on the OS. 
{{% /alert %}}
```

Reply:
What do you mean specifically by read data here, this is broad. Nearly every component reads data and do you means, reads data into memory? Without guidance this statement is not useful. For example does every pub/sub component read data? At least provide an example.

Reply:
Updated to clarify what reading data means